src/stories/Button.stories.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
```

Comment:
Looks, Button exists, it isn't visible? Please double check if you have time
![image](https://user-images.githubusercontent.com/17515584/116823176-b4f9b680-ab8b-11eb-86d0-530a625630f5.png)
![image](https://user-images.githubusercontent.com/17515584/116823191-c93db380-ab8b-11eb-8ccd-e0cb30e9da58.png)